helm/app/tests/values-jobs.yaml

```diff
@@ -0,0 +1,32 @@
+services:
+  jobs-not-specified:
+    enabled: true
+  jobs-empty:
+    enabled: true
+    jobs: {}
+  some-jobs:
+    enabled: true
+    jobs:
+      app-init:
+        annotations:
+          helm.sh/hook: "pre-install"
+          helm.sh/hook-delete-policy: "before-hook-creation"
+          argocd.argoproj.io/sync-wave: "10"
+          # an app init job doesn't need argocd hooks, as it runs once only anyway
```

Comment:
ArgoCD converts `helm.sh/hook: "pre-install"` to PreSync anyway, based on https://argo-cd.readthedocs.io/en/stable/user-guide/helm/#helm-hooks